### ISSUE TITLE
how to avoid insert the same style

### ISSUE THREAD

**Issue by zhu1033527427** [mui-org/material-ui-pickers#1259]
When @material-ui/pickers and @material-ui/core/Select in a same page，it will insert the same css，then they will be affected by each other，how to insert the style only once time!

![](http://lingshoutong.oss-cn-hangzhou.aliyuncs.com/lily/WX20190816-152028@2x.png)

![](http://lingshoutong.oss-cn-hangzhou.aliyuncs.com/lily/WX20190816-152304@2x.png)

**Comment by dmtrKovalenko** [comment]
Duplicate of #1143
